app/Http/Resources/InviteResource.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
It seems that `InviteResource` is not being used anywhere. Consider removing it.